packages/jest-haste-map/src/index.ts

```diff
@@ -326,7 +326,7 @@ class HasteMap extends EventEmitter {
     name: string,
     ...extra: Array<string>
   ): string {
-    const hash = crypto.createHash('md5').update(extra.join(''));
+    const hash = crypto.createHash('md5').update(extra.join('') + 'v2');
```

Comment:
You don't need this. The version is included in the hash and you'll get a new name when you upgrade Jest.

Reply:
Did not know! Fixed. Thanks.